.github/workflows/build-image.yml

```diff
@@ -0,0 +1,205 @@
+name: Build Container Image & Test
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    # Publish semver tags as releases.
+    tags: ["v*.*.*"]
+    paths-ignore:
+      - "README.md"
```

Comment:
No, it shouldn't

Reply:
wdym?

Reply:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushbranchestagsbranches-ignoretags-ignore

Reply:
And it worked fine in my testing before I was added as a collaborator here. (For some reason when I was added as a collaborator the action runs started linking here instead of in my repository, but when I made a new release it'd run twice)

Reply:
so will it run a push to main? my understanding is no

Reply:
yes it will

Reply:
You'd have to exclude all branches to avoid that if I'm understanding the docs correctly.

Reply:
This sentence implies otherwise:
> If you define only tags/tags-ignore or only branches/branches-ignore, the workflow won't run for events affecting the undefined Git ref.

Reply:
damn i'm dumb. Thanks for pointing that out

Reply:
Yep that's why it wasn't building in my repository.

Reply:
np its all good